resources/waterheater.rb

```diff
@@ -936,6 +935,52 @@ def self.calc_ef_from_uef(uef, type, fuel_type)
     return nil
   end
 
+  def self.calc_tank_areas(act_vol)
+    pi = Math::PI
+    height = 48.0 # inches
+    diameter = 24.0 * ((act_vol * 0.1337) / (height / 12.0 * pi))**0.5 # inches
+    a_top = pi * (diameter / 12.0)**(2.0) / 4.0 # sqft
+    a_side = pi * (diameter / 12.0) * (height / 12.0) # sqft
+    surface_area = 2.0 * a_top + a_side # sqft
+
+    return surface_area, a_side
+  end
+
+  def self.calc_indirect_tank_ua(act_vol, standby_loss, jacket_r, a_side, runner)
```

Comment:
Sorry forgot to pass in runner before. Now locally tested bad standby samples, it gave error/warning as expected.

Reply:
@shorowit Should I add a runner.nil check here to make sure energy_rating_index can call it without runner? Error-checking should be completed prior to ERI call.

Reply:
Since ERI shouldn't ever hit an error, I think it's ok to just pass nil for runner.

Reply:
I was worried by the warning instead of error condition actually.